mirdata/datasets/tiny_4mula.py

```diff
@@ -0,0 +1,319 @@
+"""4MuLA-tiny Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    This dataset is a tiny version extracted by 4MuLA dataset:
+
+    .. code-block:: latex
+
+        "4MuLA: A Multitask, Multimodal, and Multilingual Dataset of Music Lyrics and Audio Features."
+        by Angelo Cesar Mendes da Silva, Diego Furtado Silva, and Ricardo Marcondes Marcacini.
+        In Proceedings of the Brazilian Symposium on Multimedia and the Web (WebMedia '20). 2020.
+        DOI:https://doi.org/10.1145/3428658.3431089
+
+
+    The dataset consists of 1988 musics represents by melspectrogram and metadata. It
+    contains 27 genres and 93 artists. The audio file is not available. All attributes are:
+    ['music_id', 'music_name', 'music_lang', 'music_lyrics', 'art_id',
+    'art_name', 'art_rank', 'main_genre', 'related_genre', 'related_art',
+    'related_music', 'musicnn_tags', 'melspectrogram']
+
+"""
+import os
+from mirdata.download_utils import RemoteFileMetadata, downloader
+from mirdata import core, jams_utils
+from numpy import load, ndarray
+from pandas import read_csv, read_parquet
```

Comment:
We are reading `csv` files using the `csv` library. We do not have `pandas` as dependency. I recommend you load the files using `csv`, there are several examples in the library such as [this one](url).